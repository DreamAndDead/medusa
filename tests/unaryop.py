a = 45
a = -a

b = +a

c = ~a

test = True
nt = not test

print(a, b, c)
print(test, nt)