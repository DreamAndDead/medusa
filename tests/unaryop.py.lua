dofile('./pythonlua/luainit.lua')

local a = 45
a = -a
local b = a
local c = bit32.bnot(a)
local test = true
local nt = not test
print(a, b, c)
print(test, nt)