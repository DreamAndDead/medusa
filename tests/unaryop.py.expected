-45	-45	44
true	false
